### DISCUSSION TITLE
Problem with css and html href in github pages

### DISCUSSION THREAD

**Discussion by jelkeis** [community/community#54103] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi!

I uploaded a page that I made with some classmates to GitHub Pages and the CSS isn't working, in addition to that, the links that redirect to other HTML pages are also not working, so I thought it could be a problem with the file path, but I really can't find the error.
This is the github link of the page: https://github.com/jelkeis/proyectoAP/tree/gh-pages

I hope its understood correctly since my English isn't very good.

Thnks <3

**Comment by airtower-luna** [comment, marked as answer]
You are right that there's a problem with the paths. The problem is that you're using *absolute paths* instead of relative. You site is deployed at https://jelkeis.github.io/proyectoAP/, and for example the stylesheet is [included like this](https://github.com/jelkeis/proyectoAP/blob/00082efcde7e20f3994ef782f1409e9811b44af9/index.html#L21):

```html
	<link rel="stylesheet" href="/CSS/style.css">
```

Because of the slash (`/`) at the beginning, your browser looks at the path from the base of the domain (`https://jelkeis.github.io/`), which leads to `https://jelkeis.github.io/CSS/style.css`. That doesn't exist.

Ideally, you should use a relative link instead:

```html
	<link rel="stylesheet" href="CSS/style.css">
```

That will make the browser start looking from the *current directory* (where your `index.html` is).

Another option would be to include the `/proyectoAP/` part in the link, but then it'd break if you deploy it elsewhere. :sweat_smile: 

See also: https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#absolute_urls_vs._relative_urls

**Reply by jelkeis** [reply]
"You're right!!!
I knew how absolute and relative paths work, but I didn't know it worked that way on GitHub pages (referring to the explanation: https://jelkeis.github.io/CSS/style.css).

Thank you so much!!
